.github/workflows/ci.yaml

```diff
@@ -0,0 +1,34 @@
+name: CI
+on: [push, pull_request, workflow_dispatch]
```

Comment:
When do these other events occur? Push is too late, right? What triggers workflow dispatch?

Reply:
push is too late for early feedback on licenses but it's basically all we can do, so better than nothing (still catches it before releasing which is the ultimate goal). Workflow dispatch is a manual trigger, which is just a nice to have here (I've found it useful for debugging flaky tests before).